docs/Privacy/Explanation/Privacy-Groups.md

```diff
@@ -10,6 +10,13 @@ The Pantheon nodes maintain the public world state for the blockchain and a priv
 The private states contain data that is not shared in the globally replicated world state. Private transactions read 
 and write to the private world state for the privacy group, and read from the public world state.
 
+Contracts have the following levels of access between public and private states:
+
+* A contract from a privacy group can read and write to another contract in the same privacy group.
+* A contract from a privacy group can read from the public state including public contracts.
```

Comment:
> A contract from a privacy group can read from the public state including public contracts.

This is not true

Reply:
When we process the private transaction we need run everything through the evm with the private world state for that privacy group. We cannot currently access information from the public world state. To do so would require significant changes to the way we process the call stack. 

Furthermore, as it stands we can not differentiate between public and private addresses. To implement this we would probably need to implement our own opcode or perhaps do some kind of pre-compile proxy that would allow a sort of poor man's reflection.